### PR TITLE
Fixed problem dealing with text before the initial clef.

### PR DESCRIPTION
**Pull request by henryso**
Completes the fix for #873.

Please review and merge if satisfactory.

### CODE REVIEW

File: tex/gregoriotex.lua [hunk 1]
```diff
@@ -587,6 +587,7 @@ end
 local function direct_gabc(gabc, header)
   tmpname = os.tmpname()
   local f = io.open(tmpname, 'w')
+  gabc = gabc:match('^()%s*$') and '' or gabc:match('^%s*(.*%S)')
```
Comment (eroux) [inline tex/gregoriotex.lua]: I'm having difficult time undestanding what this does... could you just add a short comment above?

**Comment by eroux** [comment]
Apart from the small comment, this can be merged, thanks a lot!

**Comment by henryso** [comment]
Ok, added.

**Comment by eroux** [comment]
Thanks, merging!

**Pull request merged**